packages/raja/package.py

```diff
@@ -197,8 +197,9 @@ def initconfig_package_entries(self):
 
         entries.append(cmake_cache_option("RAJA_ENABLE_OPENMP_TASK", "+omptask" in spec))
 
+        entries.append(cmake_cache_string("BLT_CXX_STD","c++14"))
+
         if "+desul" in spec:
-            entries.append(cmake_cache_string("BLT_CXX_STD","c++14"))
```

Comment:
@rhornung67 I want your opinion on this. This was key to get RAJA to build with intel@19+gcc@8.3.1.
But the fix applies to all targets, is that a bad idea? Doesn’t RAJA require c++14 anyways?

Reply:
Yes, RAJA requires C++14. That is probably a remnant from when we required only C++11 and desul required C++14. This should apply to all targets.